### PR TITLE
add info about octane that people were asking about in Discord

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
A link and a table. 🤷‍♂

### CODE REVIEW

File: app/templates/editions/octane.hbs [hunk 1]
```diff
@@ -24,6 +24,140 @@
...
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Named Arguments</td>
+      <td></td>
+      <td>@foo vs foo vs this.foo</td>
+    </tr>
+    <tr>
+      <td>Optional Features: Application Template Wrapper &amp; Temlpate-only Glimmer Compnonents</td>
```
Comment (lupestro) [inline app/templates/editions/octane.hbs]: typo

File: app/templates/editions/octane.hbs [hunk 2]
```diff
@@ -24,6 +24,140 @@
   <li><a href="https://emberjs.github.io/rfcs/0364-roadmap-2018.html">Read The Roadmap RFC</a></li>
 </ul>
 
+<h2>Are you adventurous and want to test each of the features out?</h2>
+<p>
+  All features of Octane are presently landed in the latest version of ember,
+  but if you'd like to try pieces of Octane before upgrading all the way to 3.14+,
+  you may find the below helper: a quick reference of each of the features, and when you can use them.
```
Comment (theroncross) [inline app/templates/editions/octane.hbs]: ...below helpful: ...

File: app/styles/app.scss [hunk 3]
```diff
@@ -438,4 +438,34 @@ li[aria-expanded=true] ul[role=menu].dropdown {
   border-radius: 0;
 }
 
-/* end hotfix */
\ No newline at end of file
+/* end hotfix */
+
+.octane-feature-summary-table {
```
Comment (MelSumner) [inline app/styles/app.scss]: any table css should go to github.com/ember-learn/ember-styleguide please.

File: app/templates/editions/octane.hbs (hunk 1 above)
Comment (MelSumner) [inline app/templates/editions/octane.hbs]:
```suggestion
      <td>Optional Features: Application Template Wrapper &amp; Temlpate-only Glimmer Components</td>
```

File: app/templates/editions/octane.hbs [hunk 4]
```diff
@@ -24,6 +24,140 @@
   <li><a href="https://emberjs.github.io/rfcs/0364-roadmap-2018.html">Read The Roadmap RFC</a></li>
 </ul>
 
+<h2>Are you adventurous and want to test each of the features out?</h2>
+<p>
+  All features of Octane are presently landed in the latest version of ember,
```
Comment (MelSumner) [inline app/templates/editions/octane.hbs]:
```suggestion
  All features of Octane are presently landed in the latest version of Ember,
```

File: app/templates/editions/octane.hbs [hunk 5]
```diff
@@ -24,6 +24,140 @@
...
+      <td>Named Arguments</td>
+      <td></td>
+      <td>@foo vs foo vs this.foo</td>
+    </tr>
+    <tr>
+      <td>Optional Features: Application Template Wrapper &amp; Temlpate-only Glimmer Compnonents</td>
+      <td></td>
+      <td>Introduced with defaults: "off"</td>
+    </tr>
+    <tr class="spacer"><td colspan="4"></td></tr>
```
Comment (MelSumner) [inline app/templates/editions/octane.hbs]:
```suggestion
    <tr class="spacer"><td colspan="4"></td></tr>
```
Can you explain more about what this is?

Comment (NullVoxPopuli) [reply app/templates/editions/octane.hbs]: a way to visually separate the different versions

File: app/styles/app.scss (hunk 3 above)
Comment (NullVoxPopuli) [reply app/styles/app.scss]: what does that look like for smaller screens? does the ember-styleguide have any un-implemented tables? last I looked through the docs, there was nothing about tables?

File: app/templates/editions/octane.hbs (hunk 5 above)
Comment (MelSumner) [reply app/templates/editions/octane.hbs]: This will mess up screen readers, so let's add something with CSS instead.

**Comment by MelSumner** [comment]
@NullVoxPopuli let's work to sort out the issues in this PR or close it. Thank you! :)

**Comment by NullVoxPopuli** [comment]
Since octane's release, I haven't seen anyone ask for this info? :Shrug: :D

**Pull request closed without merging**